qtgui/SourceImage.cpp

```diff
@@ -79,6 +92,7 @@ void SourceImage::setPath(const QUrl& data)
   clipTopLeft = { 0, 0 };
   clipBottomRight = { image.width(), image.height() };
   update();
+  logging::LogStream::instance().getLogStream(logging::Level::DEBUG) << "File Size is " << image.width() << "x" << image.height() << std::endl;
```

Comment:
not so bad to log this all the time. that, however, isn't quite true for the one on MouseMoved.